docs/csharp/fundamentals/program-structure/main-command-line.md

```diff
@@ -29,33 +29,36 @@ For more information, see [Top-level statements](top-level-statements.md).
 ## Overview
```

Comment:
> The Main method is the entry point of a C# application. (Libraries and services do not require a Main method as an entry point.) 

This isn't in your change, however I'd like to suggest removing the entire parenthetical. I don't know why anyone would conclude that a description of applications would apply to libraries.

Reply:
Moreover, I found this misleading, because I code my service with a Main to facilitate the debug.